apps/web/components/manual-feedback-form.tsx

```diff
@@ -0,0 +1,52 @@
+"use client";
+
+import { z } from "zod";
+import { zodResolver } from "@hookform/resolvers/zod";
+import {
+  Form,
+  FormControl,
+  FormField,
+  FormItem,
+  FormLabel,
+  FormMessage,
+} from "@repo/ui/components/form";
+import { Textarea } from "@repo/ui/components/textarea";
+import { useForm } from "react-hook-form";
+
+const FormSchema = z.object({
+  feedbacks: z.string().min(30, {
+    message: "Bio must be at least 30 characters.",
```

Comment:
Why Bio ?

Reply:
Could be 

```tsx
const MIN_FEEDBACKS_LENGTH = 30;

const FormSchema = z.object({
  feedbacks: z.string().min(MIN_FEEDBACKS_LENGTH, {
    message: `Feedbacks must be at least ${MIN_FEEDBACKS_LENGTH} characters.`,
  }),
});
```

Reply:
😄 Sorry, it was just textarea. Future pull-request already hav better validation with correct details